.travis.yml

```diff
@@ -12,25 +12,22 @@ compiler:
     - gcc
 
 env:
+  global:
+    - MOZ_BRANCH='mozilla-central'
+    - DISPLAY=$(if [[ $TRAVIS_OS_NAME = 'linux' ]]; then echo ':99'; fi)
+    - MOZ_XVFB=$(if [[ $TRAVIS_OS_NAME = 'linux' ]]; then echo 1; fi)
```

Comment:
Lovely! This cleans up the 3rd column on Travis a lot and makes it way better readable.